.eslintrc.js

```diff
@@ -0,0 +1,215 @@
+module.exports = {
+    "env": {
+        "browser": true
+    },
+    "extends": "eslint:recommended",
+    "rules": {
+        "indent": ["error", 2],
+        "no-console":"error",
+        "accessor-pairs": "error",
+        "array-bracket-spacing": "error",
+        "array-callback-return": "error",
+        "arrow-body-style": "error",
+        "arrow-parens": "error",
+        "arrow-spacing": "error",
+        "block-scoped-var": "error",
+        "block-spacing": [
+            "error",
+            "never"
+        ],
+        "brace-style": "off",
+        "callback-return": "error",
+        "camelcase": "error",
+        "comma-dangle": [
+            "error",
+            "never"
+        ],
+        "comma-spacing": [
+            "error",
+            {
+                "after": true,
+                "before": false
+            }
+        ],
+        "comma-style": [
+            "error",
+            "last"
+        ],
+        "complexity": "error",
+        "computed-property-spacing": "error",
+        "consistent-return": "error",
+        "consistent-this": "error",
+        "curly": "error",
+        "default-case": "error",
+        "dot-location": "error",
+        "dot-notation": "error",
+        "eol-last": "error",
+        "eqeqeq": "off",
+        "func-names": "off",
+        "func-style": [
+            "error",
+            "expression"
+        ],
+        "generator-star-spacing": "error",
+        "global-require": "error",
+        "guard-for-in": "error",
+        "handle-callback-err": "error",
+        "id-blacklist": "error",
+        "id-length": [
+            "off",
+            {
+                "properties": "never"
+            }
+        ],
+        "id-match": "error",
+        "indent": "off",
```

Comment:
When you repeat rules, the last one present wins. This one turns off the indent rule you set up near the top (it's set to "off" when you want "error".)